search-enterprise/latest/admin/index.md

```diff
@@ -407,4 +449,5 @@ Main use case to reindex only Content or Path is a fully metadata indexed Reposi
 
 When indexing large repositories from scratch, metadata indexing rate will be higher than content indexing rate. This will increase the number of messages pending in `acs-repo-transform-request` ActiveMQ queue with the time. However, since [default ActiveMQ configuration](https://activemq.apache.org/amq-message-store) is prepared to handle [million of messages](https://activemq.apache.org/how-do-i-configure-activemq-to-hold-100s-of-millions-of-queue-messages) per queue, this won't be an issue for the platform. 
 
->> If you are using a custom ActiveMQ configuration, verify that messages can be persisted not only in memory but also in the filesystem.
+If you are using a custom ActiveMQ configuration, verify that messages can be persisted not only in memory but also in the filesystem.
```

Comment:
I would say something like _"If you're using a custom ActiveMQ configuration, make sure the message store makes use of paging cache (according to the documentation linked above all message stores except the memory store use the paging cache). In other words make sure ActiveMQ is not using a transient message store"_

Reply:
Adjusted the text cheers.